Library/Source/NativeEngine.cpp

```diff
@@ -275,23 +275,24 @@ namespace Babylon
             bgfx::TextureFormat::RGBA32F};
     }
 
-    Napi::FunctionReference NativeEngine::InitializeAndCreateConstructor(Napi::Env& env)
+    void NativeEngine::InitializeDeviceContext(void *nativeWindowPtr, uint32_t width, uint32_t height)
```

Comment:
The name DeviceContext is a bit overloaded, especially with DirectX. Maybe just `Initialize`?

Reply:
InitializeContext?
Initialize is a too vague

Reply:
InitializeWindow?

Reply:
Though I don't think Initialize is very vague. We are initializing bgfx inside.